src/Makefile

```diff
@@ -10,15 +10,15 @@ else
 XOBJS = $(DOBJS)
 endif
 
-CFLAGS-add += -DMATHLIB_STANDALONE
+CFLAGS-add += -std=c99 -DMATHLIB_STANDALONE
```

Comment:
I don't think it uses any of the GNU features: http://clang.llvm.org/docs/UsersManual.html#differences-between-various-standard-modes
and I thought it might make it more extensible.